package.json

```diff
@@ -45,7 +45,7 @@
     },
     "overrides": [
       {
-        "files": ["packages/terra-theme-properties/**/*.scss"],
+        "files": ["packages/terra-theme-properties/**/*.scss", "packages/**/redwood-theme/*.scss"],
```

Comment:
What rule is causing this change to be required?  I would not expect the linter to fail on the scss files for the themes.

Reply:
Good catch, fixed in https://github.com/cerner/terra-framework/pull/1994/commits/dcf6be070dc631b1296ea771e9b984a402cd7616